sympy/stats/tests/test_joint_rv.py

```diff
@@ -68,3 +69,11 @@ def test_JointRV():
     assert density(X)(1, 2) == exp(-2)/(2*pi)
     assert isinstance(X.pspace.distribution, JointDistributionHandmade)
     assert marginal_distribution(X, 0)(2) == sqrt(2)*exp(-S(1)/2)/(2*sqrt(pi))
+
+def test_expectation():
+    from sympy import simplify
+    from sympy.stats import E
+    m = Normal('A', [x, y], [[1, 0], [0, 1]])
+    assert simplify(E(m[1])) == y
+    raises (NotImplementedError, lambda: E(m))
```

Comment:
We should test correct things in the tests. If it's missing the implementation, maybe it should be in an `XFAIL` test.

Reply:
Thanks. I have updates the tests.